game-core/src/main/java/games/strategy/triplea/ui/MovePanel.java

```diff
@@ -299,20 +299,14 @@ private void selectUnitsToMove(
                       candidateAirTransportsMatch);
               // candidateAirTransports.removeAll(selectedUnits);
               candidateAirTransports.removeAll(dependentUnits.keySet());
-              if (unitsToLoad.size() > 0 && candidateAirTransports.size() > 0) {
+              if (!unitsToLoad.isEmpty() && !candidateAirTransports.isEmpty()) {
                 final Collection<Unit> airTransportsToLoad =
                     getAirTransportsToLoad(candidateAirTransports);
                 selectedUnits.addAll(airTransportsToLoad);
                 if (!airTransportsToLoad.isEmpty()) {
                   final Collection<Unit> loadedAirTransports =
                       getLoadedAirTransports(route, unitsToLoad, airTransportsToLoad, player);
                   selectedUnits.addAll(loadedAirTransports);
-                  final Map<Unit, Unit> unitsToTransports =
-                      TransportUtils.mapTransportsToLoad(loadedAirTransports, airTransportsToLoad);
-                  final MoveDescription message =
-                      new MoveDescription(
-                          loadedAirTransports, route, unitsToTransports, dependentUnits);
-                  setMoveMessage(message);
```

Comment:
Can you clarify why this can be removed?

Reply:
setMoveMessage() is for the move that will be returned via waitForMove(), but this interaction does not actually result in a move that is sent to the backend, instead this just initiates the move and the move that will be returned is when the final destination is selected. I've verified that moving paratroopers still works with this removed.

Reply:
Ok. Yeah, I looked at this code more closely and can't figure out why the move message would need set since this is just finding the selected units to move. The move message should always be set when you select an end point for the units and actually have a route.